public/keymaps/heliar_wm1_hotswap_default.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard": "heliar/wm1_hotswap",
+  "keymap": "default",
+  "commit": "95cc82d1a6c061ce7224d8893049dd0598fd66bd",
```

I use this for internal tracking, but Configurator's API doesn't use them (yet, I hope).

I get this by going to the default keymap's directory – [here, in your case](https://github.com/qmk/qmk_firmware/blob/master/keyboards/heliar/wm1_hotswap/keymaps/default/keymap.c) – and copying the full commit hash. Every once in a while I check qmk_firmware to see if any of the default keymaps have been updated, by comparing the commit hashes again.

```suggestion
  "commit": "2b318ba01b6e504bfe5f300f8459afc849e104b7",
```